Backend/TimeTracker.Api/Controllers/ProjectController.cs

```diff
@@ -59,24 +59,37 @@ public async Task<GenericResponseDTO<Project>> GetProjectById(int id)
         /// </summary>
         /// <returns>A list of all the projects the user is the teacher or is a student of</returns>
         [Authorize]
-        [HttpGet]
-        public async Task<GenericResponseDTO<List<Project>>> GetProjectsByUser()
+        [HttpGet]
+        [Route("Bool")]
```

Comment:
Can probably just remove this I don't think this needs to be `/bool`

Reply:
Removed Bool